dodiscover/replearning/gin.py

```diff
@@ -0,0 +1,115 @@
+"""
+Wrapper for the GIN latent variable causal discovery algorithm in causal-learn
+"""
+
+from pandas import DataFrame
+
+from causallearn.search.HiddenCausal.GIN.GIN import GIN as GIN_
+from pywhy_graphs import CPDAG
+
+
+class GIN:
+    """Wrapper for GIN in the causal-learn package.
+
+    Parameters
+    ----------
+    indep_test_method : str
+        The method to use for testing independence, by default "kci"
+    alpha : float
+        The significance level for independence tests, by default 0.05
+
+    Attributes
+    ----------
+    graph_ : CPDAG
+        The estimated causal graph.
+    causal_learn_graph : CausalGraph
```

Comment:
```suggestion
    causal_learn_graph_ : CausalGraph
```

Since these are fitted attributes, should also have trailing `_`

Reply:
Is this a scikitlearn interface thing?  Can you explain more?